src/components/Popup.tsx

```diff
@@ -18,13 +19,15 @@ const Popup = React.forwardRef<
   PropsWithChildren<{
     // used to cancel imports
     importFileId?: string
+    // used to cancel multicursor alerts
+    multicursor?: boolean
```

Comment:
Can you get this from `state.alert.alertType` rather than passing it in as a prop?